master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -33,6 +33,11 @@ kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/instal
 
 >[Click here to view the above yaml directly.](1.5/calico.yaml)
 
+## Using calicoctl in a Kubeadm Cluster
+
+The simplest way to use calicoctl in Kubeadm is by running it as a pod.
+See [Part b. of Calico's Guide on Using Calicoctl in Kubernetes](../../../tutorials/using-calicoctl#b-running-calicoctl-as-a-kubernetes-pod) for more information.
```

Comment:
I think the link here can be simplified - it doesn't need to say "part b of Calico's guide" (we're linking directly to that section, and no need to specify that it belongs to Calico)

We should use the actual title of the doc - how 'bout:

```
See [running calicoctl on Kubernetes] for more information.
```

Or,

```
See [running calicoctl as a Kubernetes Pod] for more information.
```